client/my-sites/sharing/connections/services/eventbrite.js

```diff
@@ -1,61 +1,48 @@
 /**
  * External dependencies
  */
-var last = require( 'lodash/last' );
+import { last } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var connectionsList = require( 'lib/connections-list' )(),
-	sites = require( 'lib/sites-list' )();
+import connectionsList from 'lib/connections-list';
+import sites from 'lib/sites-list';
```

Comment:
You can create a component that renders nothing like we do for QueryComponents or if it doesn't make sense we can update the interfaces to pass in a site.